tests/services/test_snippet.py

```diff
@@ -121,6 +121,49 @@ class TestSnippet(metaclass=AIOTestMeta):
         excinfo.match(r'Sorry, cannot complete the request since `marker` '
                       r'points to a nonexistent snippet.')
 
+    async def test_get_filter_by_title(self):
+        result = await self.service.get(title='snippet #1')
+        assert result == [self.snippets[0]]
+
+        prefix_match = await self.service.get(title='snippet')
+        assert prefix_match == list(reversed(self.snippets))
+
+        regexes_are_escaped = await self.service.get(title='^snippet.*')
+        assert regexes_are_escaped == []
+
+        nonexistent = await self.service.get(title='non existing snippet')
+        assert nonexistent == []
+
+    async def test_get_filter_by_tag(self):
+        result = await self.service.get(tag='tag_c')
```

Comment:
Maybe add also a test to ensure that the filtering works even for snippets with few tags?

Reply:
Done.